### PR TITLE
Create prefix of file using ansible.builtin.tempfile

### PR DESCRIPTION
**Pull request by theputta**
##### SUMMARY

Prefix of file name created by module using ansible.builtin.tempfile

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: lib/ansible/modules/tempfile.py [hunk 1]
```diff
@@ -66,6 +66,12 @@
     state: file
     suffix: temp
   register: tempfile_1
+  
+- name: Create Prefix of file
```
Comment (Akasurde) [inline lib/ansible/modules/tempfile.py]:
```suggestion
- name: Create a temporary file with a specific prefix
```

File: lib/ansible/modules/tempfile.py [hunk 2]
```diff
@@ -66,6 +66,12 @@
     state: file
     suffix: temp
   register: tempfile_1
+
```
Comment (Akasurde) [inline lib/ansible/modules/tempfile.py]: ERROR: lib/ansible/modules/tempfile.py:69:1: W293: blank line contains whitespace (100%)

Comment (theputta) [reply lib/ansible/modules/tempfile.py]: I am trying to use that as a differentiator from the next example

Comment (theputta) [reply lib/ansible/modules/tempfile.py]: @Akasurde

- Sorry, initially dint understand what you meant. 
- Removed the white spaces and pushed the changes for your review.

**Pull request merged**